### PR TITLE
ProcessingSplash: new design

### PR DESCRIPTION
**Pull request by selsta**
Screenshot 2020-04-27 at 01 36 26
Screenshot 2020-04-27 at 01 33 21

### CODE REVIEW

File: components/ProcessingSplash.qml [hunk 1]
```diff
@@ -56,44 +58,57 @@ Rectangle {
...
         }
 
 
         MoneroComponents.TextPlain {
-            id: heightProgress
-            font {
-                pixelSize: 18
-            }
+            id: messageTitle
+            text: "Please wait..."
```
Comment (xiphon) [inline components/ProcessingSplash.qml:105]: Translation

File: components/Style.qml [hunk 2]
```diff
@@ -58,6 +58,7 @@ QtObject {
     property string titleBarCloseSource: blackTheme ? _b_titleBarCloseSource : _w_titleBarCloseSource
     property string titleBarButtonHoverColor: blackTheme ? _b_titleBarButtonHoverColor : _w_titleBarButtonHoverColor
 
+    property string moneroLogoSource: "qrc:///images/monero-vector.svg"
```
Comment (xiphon) [inline components/Style.qml:61]: ?

File: components/ProcessingSplash.qml (hunk 1 above)
Comment (selsta) [reply components/ProcessingSplash.qml]: updated

File: components/Style.qml (hunk 2 above)
Comment (selsta) [reply components/Style.qml]: updated

wasn’t sure where to add it, I skipped an extra Style.qml entry now

File: components/ProcessingSplash.qml [hunk 3]
```diff
@@ -29,21 +29,23 @@
...
 import QtQuick.Layouts 1.1
 
 import "../components" as MoneroComponents
 
 Rectangle {
     id: root
-    color: MoneroComponents.Style.blackTheme ? "white" : "transparent"
+    color: MoneroComponents.Style.blackTheme ? "black" : "white"
     visible: false
+    radius: 10
```
Comment (xiphon) [inline components/ProcessingSplash.qml:41]: Looks inconsistent with the other dialogs

Comment (selsta) [reply components/ProcessingSplash.qml:41]: Plan is to update the other dialogs too over time:

https://www.figma.com/file/DplJ2DDQfIKiuRvolHX2hN/Monero-GUI?node-id=2%3A4925

Comment (xiphon) [reply components/ProcessingSplash.qml:41]: Ok, let's use radius.

**Comment by selsta** [comment]
@GBKS Can you upload the spinner from Figma as an .svg?

**Comment by GBKS** [comment]
SVG can't do conic gradients unfortunately, would a PNG do as well? I'm attaching 1x and 2x versions here. 
load-spinner
load-spinner@2x

**Comment by selsta** [comment]
@GBKS Could you upload the 1x version with 80x80 pixels (and the corresponding 2x)? Thanks!

**Comment by selsta** [comment]
@GBKS The 66x66 current size is okay! No need for a new upload. Thanks again.

**Comment by selsta** [comment]
@GBKS New problem now, can you check if the busy indicator is a proper circle? It does not seems so.
Screenshot 2020-04-29 at 09 39 59
The red line is taken from above, it should be the same size as the top margin but the bottom margin between logo and busy indicator is smaller.

**Comment by GBKS** [comment]
Good catch. The width was off by one pixel. Here are the updated files.

load-spinner
load-spinner@2x

**Comment by selsta** [comment]
@GBKS thanks, the new ones are centred now.

Updated.

**Comment by GBKS** [comment]
Woohoo!

**Pull request merged**